tox.ini

```diff
@@ -9,22 +11,29 @@ deps =
     readthedocs-sphinx-ext
     pytest
     sphinxcontrib-httpdomain
-    sphinx16: Sphinx < 1.7
-    sphinx17: Sphinx < 1.8
-    sphinx18: Sphinx < 1.9
-    sphinx20: Sphinx < 2.1
-    sphinx21: Sphinx < 2.2
-    sphinx22: Sphinx < 2.3
-    sphinx23: Sphinx < 2.4
-    sphinx24: Sphinx < 2.5
-    sphinx30: Sphinx < 3.1
-    sphinx31: Sphinx < 3.2
-    sphinx32: Sphinx < 3.3
-    sphinx33: Sphinx < 3.4
-    sphinx34: Sphinx < 3.5
-    sphinx35: Sphinx < 3.6
-    sphinx40: Sphinx < 4.1
-    sphinx41: Sphinx < 4.2
+    sphinx16: Sphinx>=1.6,<1.7
```

Comment:
I'm curious what the lower bound additions are for. Was Sphinx being downgraded by another dependency?

Reply:
I guessed from the `sphinx16` name that it should target Sphinx 1.6, do we want builds for lower versions and should we add the explicitly tagged?